src/content/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent.mdx

```diff
@@ -0,0 +1,181 @@
+
+1. Install our Kubernetes integration. See how to install it [here](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure).
+
+<Callout variant="tip">
+  Alternatively, we also offer fully manual instructions for [deploying our integration using Helm](/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm).
+</Callout>
+
+2. To ensure the integration has been configured correctly, go to **[one.newrelic.com](https://one.newrelic.com) > All capabilities > Query builder**, and run this NRQL query to see if data has been reported:
+
+  ```sql
+  FROM Metric SELECT count(*) WHERE collector.name = 'prometheus-agent' AND cluster_name = 'YOUR_CLUSTER_NAME' since 1 hour ago
+  ```
+
+<Callout variant="tip">
+  If you don't see your data right away, wait for a few seconds. Data may take some time to make it to New Relic.
+</Callout>
+
+By default, the New Relic Prometheus agent uses two annotations to discover targets: `newrelic.io/scrape: "true"` and `prometheus.io/scrape: "true"`. All the targets in the cluster that are annotated with `newrelic.io/scrape: "true"` are discovered and scraped. However, the default annotation from the Prometheus community, `prometheus.io/scrape: "true"`, is discovered but we filter them to only scrape the [Prometheus integrations](/docs/infrastructure/prometheus-integrations/integrations-list/integrations-list-intro) that we support.
+
+[Prometheus Integrations](/docs/infrastructure/prometheus-integrations/integrations-list/integrations-list-intro) help you get curated experiences for services that expose metrics in Prometheus or Open Metrics format, bundling everything you need to start monitoring out of the box.
+
+For more information, see the [set up New Relic's Prometheus agent](/docs/integrations/prometheus-integrations/install-configure-prometheus-agent/setup-prometheus-agent) documentation.
+
+## Install only the Prometheus agent [#prometheus-install]
+
+If you don't need the Kubernetes integration, you can install the Prometheus agent by its own.
+
+1. Install the [Prometheus Agent](https://github.com/newrelic/newrelic-prometheus-configurator#readme) by running:
+  ```shell
+  helm repo add newrelic-prometheus https://newrelic.github.io/newrelic-prometheus-configurator
+  helm upgrade --install newrelic newrelic-prometheus/newrelic-prometheus-agent -f YOUR_CUSTOM_VALUES.yaml
+  ```
+
+2. To ensure the integration has been configured correctly, go to **[one.newrelic.com](https://one.newrelic.com) > All capabilities > Query builder**, and run this NRQL query to see if data has been reported:
+
+  ```sql
+  FROM Metric SELECT count(*) WHERE collector.name = 'prometheus-agent' since 1 hour ago
+  ```
+
+  <figcaption>
+    Install the quickstart containing the [dashboard for the Prometheus agent](https://newrelic.com/instant-observability/prometheus-agent).
+  </figcaption>
+
```

Comment:
I believe this is here by mistake, there is the very same section 10 lines below